torch/csrc/distributed/rpc/request_callback_no_python.cpp

```diff
@@ -685,6 +685,33 @@ bool RequestCallbackNoPython::cudaAvailable() const {
 #endif
 }
 
+c10::intrusive_ptr<JitFuture> RequestCallbackNoPython::asFuture(
```

Comment:
Any reason these need to be member functions instead of standalone util functions?

Reply:
No "strong" reason, but:
- several other helper functions in this file were defined as methods (e.g., `cudaAvailable`)
- we need this to be available both to `RequestCallbackNoPython` and to its subclass `RequestCallbackImpl` (hence it needs to be public, it cannot go in an unnamed namespace)
- by making it a method in a sense we "namespace" it